lib/puppet/provider/cisco_ace/nxapi.rb

```diff
@@ -1,9 +1,9 @@
 #
 # The NXAPI provider for cisco_ace.
 #
-# July, 2015
+# January 2016
```

Comment:
No, I think this is noise

From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-puppet-module <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, January 25, 2016 at 2:38 PM
To: cisco/cisco-network-puppet-module <cisco-network-puppet-module@noreply.github.com<mailto:cisco-network-puppet-module@noreply.github.com>>
Cc: Cisco Employee <dcherian@cisco.com<mailto:dcherian@cisco.com>>
Subject: Re: [cisco-network-puppet-module] vPC documentation in README.md (#146)


In lib/puppet/provider/cisco_ace/nxapi.rb<https://github.com/cisco/cisco-network-puppet-module/pull/146#discussion_r50768351>:

> @@ -1,9 +1,9 @@
>  #
>  # The NXAPI provider for cisco_ace.
>  #
> -# July, 2015
> +# January 2016


Why do you have cisco_ace diffs here? Was this merge noise?

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-puppet-module/pull/146/files#r50768351>.